types/knuddels-userapp-backend-api/index.d.ts

```diff
@@ -145,18 +145,33 @@ declare global {
         /**
          * @see https://developer.knuddels.de/docs/classes/App.html#property_chatCommands
          */
-        chatCommands?: { [commandName: string]: (user: User, params: string, command: string) => void } | undefined;
```

Comment:
I'm guessing the files are generated as part of an internal process and uploaded occasionally.  If that's the case, you might want to consider having your generator add the `| undefined` to any optional properties for the benefit of any consumers using [`exactOptionalPropertyTypes`](https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes)